### ISSUE TITLE
Descriptions improvements in the Moodle Plugins directory

### ISSUE THREAD

**Issue by mudrd8mz** [turnitin/moodle-plagiarism_turnitin#454]
The plugin's description at https://moodle.org/plugins/plagiarism_turnitin could be improved.

The short description currently reads

> Turnitin is a commercial plagiarism detection system which requires a paid subscription to use - This Plugin integrates with the existing Moodle Assignment module.

It only mentions the Assignment module. But the plugin also provides integration with the Workshop or Forum.

The long description could be a bit elaborated, too. It could describe the plugin's features for a user who does not yet have any experience with TII services.

In parallel, the screenshots section could show how the plugin integration looks in the other modules (Forum or Workshop), too.

**Comment by carl-hostrander** [comment]
Thank you for reporting this.  Because the latest version of the plagiarism plugin is supported for versions of Moodle 4.1 and higher, I am closing this ticket.  However, if you find this issue is occurring with the latest version of the plugin in any of the supported Moodle versions, please create a new ticket and we will address it.
